Source/WTF/wtf/Threading.cpp

```diff
@@ -500,6 +500,9 @@ void initialize()
 {
     static std::once_flag onceKey;
     std::call_once(onceKey, [] {
+#if ENABLE(CONJECTURE_ASSERT)
+        wtfConjectureAssertIsEnabled = !!getenv("ENABLE_WEBKIT_CONJECTURE_ASSERT");
```

Comment:
Do environment variables get propagated to child processes of all kinds, or do we need to propagate them from UI process ourselves?

Reply:
I don't think we should be in the business of propagating environment variables.  The user needs to know how to set `__XPC_` environment variables if they want to target XPC processes.

Reply:
I've confirmed that `__XPC_` will do the job of passing the env var on.